staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -447,6 +457,33 @@ func (c *policyController) latestPolicyData() []policyData {
 	return res
 }
 
+func v1alphaValidationsToCELValidations(inputValidations []v1alpha1.Validation) []cel.CELValidation {
+	celValidations := make([]cel.CELValidation, len(inputValidations))
+	for i, validation := range inputValidations {
+		celValidations[i] = cel.CELValidation{
+			Expression: validation.Expression,
+			Message:    validation.Message,
+			Reason:     validation.Reason,
+		}
+	}
+	return celValidations
+}
+
+func v1alphaFailPolicyTov1FailPolicy(failPolicy *v1alpha1.FailurePolicyType) *v1.FailurePolicyType {
```

Comment:
Would have type alias from API level help us avoid conversion here?

Reply:
Could you elaborate on this please?

Reply:
Instead of redefine failurePolicy type here:
https://github.com/kubernetes/kubernetes/blob/173dde2a8c4b67027b6afcaeaf8d7e65cacc62c9/staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go#L44
make it a alias of the v1 FailurePolicyType: 
```
 type FailurePolicyType = v1.FailurePolicyType
```
It might help us avoiding unnecessary type conversion :)

Reply:
Got it!